interpreter/mesh/src/main/scala/io/buoyant/interpreter/mesh/Client.scala

```diff
@@ -305,7 +304,7 @@ object Client {
    * gRPC stream, publishing updates into the Activity's state.
    *
    * When the stream returns an error, the state is updated to
-   * Activity.Failed and no further updates will will be published.
+   * Activity. Failed and no further updates will be published.
```

Comment:
`Failed` is a member of `Activity` so I think this was correct without the space: `Activity.Failed`.

Reply:
> `Failed` is a member of `Activity` so I think this was correct without the space: `Activity.Failed`.

Thanks you for your comment. I have push a change to revert it :)